### ISSUE TITLE
Remove titlebar but keep rounded corners 

### ISSUE THREAD

**Issue by A125X** [macvim-dev/macvim#1422]
Is there any way to remove window titlebar but keep standard rounded corners? Current solution with MMNoTitleBarWindow leaves sharp and ugly corners. 

It'd be nice to see something like this instead:
<img width="643" alt="image" src="https://github.com/macvim-dev/macvim/assets/91656458/841c2213-9ee4-44bb-8a59-5d37fb88c295">

**Comment by ychin** [comment]
Not right now. What exactly do you hope to see? A thinner title bar, or just MacVim without title bar but still show a draggable window with close/minimize buttons?

**Comment by A125X** [comment]
No, I mean no title bar at all without any buttons, something like kitty terminal with hide_window_decorations titlebar-only setting written in config. 

Like on this screenshot, kitty terminal does not have any titlebar decorations and buttons, while macvim have them.
<img width="1172" alt="image" src="https://github.com/macvim-dev/macvim/assets/91656458/5195ea19-2d65-4007-bb97-e364563d9e8f">
If I want to disable them, the only option is to disable title bar completely, but it looks pretty bad because macvim becomes the only app without standard rounded corners. 
<img width="1161" alt="image" src="https://github.com/macvim-dev/macvim/assets/91656458/a257cd48-2808-4ba5-8472-128c22710f15">
It's even more noticeable when I'm using window border based focus in the tiling window manager:
<img width="650" alt="image" src="https://github.com/macvim-dev/macvim/assets/91656458/c08ac558-9983-4044-9287-333adeb2e2de">

**Comment by ychin** [comment]
Hmm, in Kitty, when you turn on that setting, is the top part (where the title bar is) draggable?

**Comment by A125X** [comment]
Seems like it's not draggable. It's still possible to resize the window, but probably there's no way to move it around without some kind of third-party window manager, I assume.

**Comment by ychin** [comment]
Right. So basically the only difference is that it has rounded corners. I took a look and have a rough idea how to implement it. Will get to it a little bit later when I have time.

**Comment by A125X** [comment]
Thanks for the support!

**Comment by winstongrace** [comment]
> Seems like it's not draggable. It's still possible to resize the window, but there's no way to move it around without some kind of third-party window manager, I assume.

Long-time user of both MacVim and Kitty. Just wanted to note that the window in Kitty is still draggable. If you hover on the left or right edge, and drag vertically, instead of horizontally to resize - it will drag the window.

Likewise, hover over the top or bottom window edge and drag horizontally.

**Comment by winstongrace** [comment]
I think it's a standard Mac thing – I can do the same with the majority of my apps. It just doesn't look like you can do it because the cursor switches to resizing arrows.
